src/tests/utils/setup.ts

```diff
@@ -12,13 +12,21 @@ import API from '@presentation/index.js';
 import { beforeAll, afterAll } from 'vitest';
 import type Api from '@presentation/api.interface';
 
+// import { readFileSync } from 'fs';
+
 /**
  * Tests setup maximum duration.
  * Added as default 10000 is not enough
  */
 const TIMEOUT = 200000;
 
 declare global {
```

Comment:
this declaration is taking a big part of the file, let's move it to separate types file maybe?

Reply:
done